circuits/src/transaction.rs

```diff
@@ -137,16 +138,18 @@ struct WitnessTxOutputNote {
 }
 
 impl TxOutputNote {
-    /// Create a tx output note
-    pub fn new(
-        note: &Note,
-        vk: &ViewKey,
-    ) -> Result<crate::transaction::TxOutputNote, PhoenixError> {
-        Ok(crate::transaction::TxOutputNote {
-            value: note.value(Some(vk))?,
-            value_commitment: note.value_commitment().into(),
-            blinding_factor: note.blinding_factor(Some(vk))?,
-        })
+    /// Crate a new `TxOutputNote`.
+    pub fn new(value: u64, blinding_factor: JubJubScalar) -> Self {
+        let value_commitment = JubJubScalar::from(value);
+        let value_commitment = (GENERATOR_EXTENDED * value_commitment)
+            + (GENERATOR_NUMS_EXTENDED * blinding_factor);
+        let value_commitment = JubJubAffine::from(value_commitment);
```

Comment:
Not sure if I'm a fan of all this variable shadowing here, but if we do this more often ignore this comment.

Reply:
Good point, fixed it